config.go

```diff
@@ -443,6 +443,8 @@ type Config struct {
 	//
 	// Optional: defaults to 200
 	DataIterationBatchSize uint64
+	// Optional: Map of schemaName => tableName => uint64 to override default values for certain tables
```

Comment:
We should document that the schemaName here is the "source" schema name.

Reply:
ok, added here https://github.com/Shopify/ghostferry/pull/246